doc/tutorials/building_acrn_in_docker.rst

```diff
@@ -0,0 +1,157 @@
+.. _building-acrn-in-docker:
+
+Building ACRN in Docker
+#######################
+
+In this tutorial, we will introduce how to build ACRN in Clear Linux Docker image.
+
+Install Docker
+**************
+
+#. Install Docker according to `Docker installation instructions <https://docs.docker.com/install/>`_.
+#. If you are behind HTTP or HTTPS proxy server, follow these
+   `HTTP/HTTPS proxy instructions <https://docs.docker.com/config/daemon/systemd/#httphttps-proxy>`_
+   to set up HTTP/HTTPS proxy to pull Docker image and
+   `Configure Docker to use a proxy server <https://docs.docker.com/network/proxy/>`_
+   to set up HTTP/HTTPS proxy for the Docker container.
+
+Get Docker Image
+****************
+
+This tutorial present two ways to get Clear Linux Docker image to build ACRN.
+
+Build Docker Image from Dockerfile
+==================================
+
+#. Download `Dockerfile <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/tutorials/Dockerfile>`_
+   to your development machine
+#. Build Docker image
+
+   .. code-block:: none
+
+      $ docker build -t acrn/clearlinux:latest -f <path/to/Dockerfile> .
```

Comment:
I would suggest to use a different tag in this example, to differentiate it from the pre-built Docker image we provide. I also like to use tags that are somewhat meaningful, so in this case, I'd call it something like `clearlinux-acrn-builder`.